ow_plexil/src/plans/Stow.plp

```diff
@@ -9,6 +9,5 @@
 
 Stow:
 {
-  SynchronousCommand unstow();
```

Comment:
I don't think this change should be in this branch, as it's part of a different PR.   I'll review that one next.

Reply:
I agree, I will remove it. I just found it helpful for speeding up testing. Apologies for the confusion.